jwpincus/punstartr
# Upvote feature

#### What does  this PR do?
Finished the upvote feature using ajax

#### Where should the reviewer start?
Review the code in votes.js, the project show page, and both tests for the vote feature (feature test and request spec that sets up the internal endpoints being consumed by the feature).

#### How should this be manually tested?
Fire up the server and manually verify that upvotes are saved for logged-in users and the user can remove them.

#### Any background context you want to provide?
This was awesome and hard!

#### What are the relevant tickets?
#146482993

#### Screenshots (if appropriate)

#### Questions:
  - Do Migrations Need to be ran? YES and seed the db
  - Do Environment Variables need to be set? no
  - Any other deploy steps?
